schema.sql

```diff
@@ -622,6 +623,7 @@ CREATE OR REPLACE PROCEDURE user_profile_update(
     userAvatar VARCHAR(128),
     notificationsEnabled BOOLEAN,
     userCountry VARCHAR(2),
+    userLocale VARCHAR(2),
```

Comment:
Since you are adding an input param you'll have to add a DROP PROCEDURE IF EXISTS for the original param list before it, should be a few examples in the schema.sql of this behavior.

Reply:
Done